var/spack/repos/builtin/packages/expat/package.py

```diff
@@ -24,19 +24,19 @@
 ##############################################################################
 from spack import *
 
+
 class Expat(Package):
     """<eXpat/> is an XML parser library written in C"""
     homepage = "http://expat.sourceforge.net/"
-    url      = "http://downloads.sourceforge.net/project/expat/expat/2.1.0/expat-2.1.0.tar.gz"
-
-    version('2.1.0', 'dd7dab7a5fea97d2a6a43f511449b7cd')
 
-    depends_on('cmake', type='build')
+    version('2.2.0', '2f47841c829facb346eb6e3fab5212e2',
+            url="http://downloads.sourceforge.net/project/expat/expat/2.2.0/expat-2.2.0.tar.bz2")
+    version('2.1.0', 'dd7dab7a5fea97d2a6a43f511449b7cd',
+            url="http://downloads.sourceforge.net/project/expat/expat/2.1.0/expat-2.1.0.tar.gz")
```

Comment:
From what I can tell, version 2.2.0 only has a bzip2 tarball and 2.1.0 only has a gzip tarball, hence the different URLs.